opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -574,9 +576,6 @@ namespace Opm
     {
         // TODO: only_wells should be put back to save some computation
         // for example, the matrices B C does not need to update if only_wells
-
-        checkWellOperability(ebosSimulator, well_state, deferred_logger);
-
         if (!this->isOperable() && !this->wellIsStopped()) return;
```

Comment:
If there is nothing changed related to operability, you probably want to move this line together.

Reply:
Since assembleWellEqWithoutIteration can be called directly and not only from assembleWellEq I think it makes sense to add the operability guard here.

Reply:
Okay. That is sensible regarding to this aspect, to have this line to use function  `assembleWellEqWithoutIteration` by itself. 

Then how about adding this  line after `checkWellOperability` in function `assembleWellEq`. It is clear that if the well is not operable, then nothing afterwards is needed in the function `assebleWellEq`. 

No strong opinion here. Just let know whether you think it is sensible.  It is merge-able either way to me.

Reply:
I dont think it is necessary since `assembleWellEq` just calls `assembleWellEqWithoutIteration` or `iterateWellEquations` which also starts with calling  `assembleWellEqWithoutIteration`.

Reply:
Okay. I just thought code will be easier to understand/read. Anyway, I will merge this.